test/integration/roles/test_ec2_eip/tasks/main.yml

```diff
@@ -152,7 +152,7 @@
 - name: assert new EIP was assigned
   assert:
     that:
-      - '"failed" not in result'
```

Comment:
this should actually be `not result|failed` or `not result.failed`, idem for most of the following